chcemvediet/apps/inforequests/tests/__init__.py

```diff
@@ -21,7 +21,7 @@
 class InforequestsTestCaseMixin(ChcemvedietTestCaseMixin):
 
     @contextlib.contextmanager
-    def assertQueriesDuringRender(self, *patterns, **kwargs):
+    def assertQueriesDuringRender(self, patterns, **kwargs):
```

Comment:
Toto je pravdepodobne preklep. 
Povodne pri volani funkcie bolo vo vsetkych troch pripadoch `len(patterns) ==  1`, pretoze to bol tuple s jednym objektom.
```
self.assertQueriesDuringRender([])
self.assertQueriesDuringRender([u'FROM "accounts_profile"'])
self.assertQueriesDuringRender([
                u'FROM "inforequests_branch" WHERE .* "inforequests_branch"."advanced_by_id" IS NULL',
                u'FROM "obligees_historicalobligee"',
                ])
```

Reply:
@viliambalaz Ak si spravne pamatam, tak spravne ma byt `*patterns`. Aby to bol tuple s polami. Kazdy prvok tuple zodpoveda jednemu volaniu `mock_render` resp. `mock_render_to_string`. Pricom kazdy taky prvok tuple obsahuje pole queries, ktore sa pocas `mock_render` resp. `mock_render_to_string` vykonali. 

Vid aj v kode metody:
```
for render_queries, render_patterns in zip(queries, patterns):
  ...
  for query, pattern in zip(render_queries, render_patterns):
    ...
```
Su tam vnorene cykly. Prvy vybera prvky z tuple, a druhy jenotlive queries z pola, ktore je prvkom tuple. Teraz ti to v `test_related_models_are_prefetched_before_render` zbehlo, pretoze `[]` je prazdne pole a teda cyklus sa nevykonal vobec.

Akurat nechapem, preco assert `self.assertEqual(len(queries), len(patterns))` ti presiel. Kedze ako `patterns` mas `[]`, co ma dlzku nula. A `len(queries)` by malo zodpovedat poctu zavolani `mock_render` resp. `mock_render_to_string`. To skor vyzera, ze `mock_render` resp. `mock_render_to_string` sa ti vobec nevolaju a mockovanie nefunguje.